akka-http/src/main/scala/akka/http/scaladsl/server/RejectionHandler.scala

```diff
@@ -165,7 +165,11 @@ object RejectionHandler {
       }
       .handle {
         case TooManyRangesRejection(_) ⇒
-          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges."))
+          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges"))
+      }
+      .handle {
+        case CircuitBreakerOpenRejection(_) ⇒
+          complete((EnhanceYourCalm, "Circuit Breaker is open; calls are failing fast"))
```

Comment:
`ServiceUnavailable`, that's what it's for. It also somewhat implies that the request hasn't yet caused any side-effects, which means it's safe to re-try.